dashboard/src/shared/RpcError.ts

```diff
@@ -0,0 +1,24 @@
+export class RpcError extends Error {
```

Comment:
I wonder if we should rename `Rpc` to `GRpc` instead

Reply:
Yeah, I thought about it, but error message states `rpc error:` so I went ahead with a generic `rpc` naming for everything.